junifer/api/utils.py

```diff
@@ -9,10 +9,19 @@
 from importlib.metadata import distribution
 from typing import Dict
 
+from ruamel.yaml import YAML
+
 from .._version import __version__
 from ..utils.logging import get_versions
 
 
+# Configure YAML class once for further use
+yaml = YAML()
+yaml.default_flow_style = False
+yaml.allow_unicode = True
+yaml.indent(mapping=2, sequence=4, offset=2)
```

Comment:
So this is a global object that you import elsewhere to load files?

Reply:
Yeah cause now you gotta have an instance of `YAML` class so doesn't make sense to create multiple objects if you have the same thing everywhere.